tapdb/assets_store.go

```diff
@@ -1055,6 +1068,18 @@ func (a *AssetStore) QueryAssetBalancesByGroup(ctx context.Context,
 		},
 	}
 
+	// We exclude the assets that are specifically used for funding custom
+	// channels. The balance of those assets is reported through lnd channel
+	// balance. Those assets are identified by the funding script tree for a
+	// custom channel asset-level script key.
+	excludeKeyStr := "0350aaeb166f4234650d84a2d8a130987aeaf6950206e090540" +
```

Comment:
We should try to derive this from `newFundingScriptTree()` instead. See my other comment on how to do that without getting into package dependency hell.

Reply:
If we do that, we should add a unit test whether `newFundingScriptTree()` returns that value though. We have to be sure that value does not accidentally change, don't you think?